index.html

```diff
@@ -143,7 +143,19 @@ <h3 class="social-media">LET’S CONNECT</h3>
         </ul>
       </section>
     </section>
-    
+    <section class="form-section">
+      <h2>Contact me</h2>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      <form action="https://formspree.io/f/xzbqzgww" class="info-form" method="post">
+        <input type="text" placeholder="Yeremias" required class="input-text" maxlength="30">
+        <input type="email" placeholder="natajayanj14@gmail.com" class="input-text" maxlength="30" required>
+        <textarea name="form-description" id="form-text-area" cols="30" rows="8" maxlength="500" placeholder="Write your message here"></textarea>
+       <button class="form-button">Get It Touch</button>
```

Comment:
Good job implementing the form 👍, everything works as expected. However, according to the requirement the various state of the button should be styled to make the button more interactive and to signal to the user what state the button is in. With this in mind, I suggest that you implement styling for the hover` and  `active` states for the button.

Although the form is getting submitted when the button is clicked, as a best practice it is advisable to add the attribute of `type="submit"`. Kindly implement this. 


<img width="629" alt="fk" src="https://user-images.githubusercontent.com/84930355/213037872-d569f4e7-6ecf-420b-a52e-9f1aafe3e01a.png">